src/main/js/controls/Bubble/helpers/helpersMultipleDataset.js

```diff
@@ -8,15 +8,12 @@ import constants from "../../../helpers/constants";
 import styles from "../../../helpers/styles";
 import utils from "../../../helpers/utils";
 import {
-    transformPoint,
+    areWeightsDefined,
     dataPointActionHandler,
     getDataPointValues,
-    decideRadius
```

Comment:
Why did we pull this out of the helpers file and duplicate it in two places?

Reply:
Because this function had references for the `maxRadius` property. I left it as it is in the original helpers file so the original Bubble API is unchanged. The duplicated function has updated property name for the new APIs.

Reply:
I think that works for now but when we get to the point of deprecating the original Bubble graph in the next major release, it would be good to move this back into the common file and replace the existing one so that we only have one instance that both bubble APIs leverage.